### PR TITLE
[Translator][Logging] implement TranslatorBagInterface.

### PR DESCRIPTION
**Pull request by aitboudad**
| Q | A |
| --- | --- |
| Bug fix? | yes |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Fixed tickets | #13632 |
| Tests pass? | yes |
| License | MIT |

### CODE REVIEW

File: src/Symfony/Component/Translation/LoggingTranslator.php [hunk 1]
```diff
@@ -85,6 +85,14 @@ public function getLocale()
     }
 
     /**
+     * {@inheritdoc}
+     */
+    public function getCatalogue($locale = null)
+    {
+        return $this->translator->getCatalogue($locale);
```
Comment (stof) [inline src/Symfony/Component/Translation/LoggingTranslator.php]: you need to check whether the inner translator implements the interface here

Comment (aitboudad) [inline src/Symfony/Component/Translation/LoggingTranslator.php]: see https://github.com/aitboudad/symfony/blob/issue_13632/src/Symfony/Component/Translation/LoggingTranslator.php#L37

Comment (stof) [inline src/Symfony/Component/Translation/LoggingTranslator.php]: hmm, indeed. I forgot it

**Comment by stof** [comment]
:+1:

**Comment by fabpot** [comment]
Thank you @aitboudad.

**Pull request merged**